pkgs/tools/security/enum4linux/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, samba, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "enum4linux";
+  version = "0.8.9";
+  src = fetchurl {
+    url = https://labs.portcullis.co.uk/download/enum4linux-0.8.9.tar.gz;
+    sha256 = "41334df0cb1ba82db9e3212981340372bb355a8160073331d2a1610908a62d85";
+  };
+
+  phases = "unpackPhase installPhase fixupPhase";
+
+  buildInputs = [samba perl];
+
+  unpackPhase = ''
+    tar xzf ${src}
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp enum4linux-0.8.9/enum4linux.pl $out/bin/enum4linux
+    chmod +x $out/bin/enum4linux
+  '';
+
+  meta = {
```

Comment:
I'm always partial to `meta = with stdenv.lib; {` and then omitting all the extra `stdenv.lib` bits.